RxSwift/Observable+Concurrency.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  Observable+Concurrency.swift
+//  RxSwift
+//
+//  Created by Shai Mishali on 22/09/2021.
+//  Copyright © 2021 Krunoslav Zaher. All rights reserved.
+//
+
+#if swift(>=5.5) && canImport(_Concurrency)
+import Foundation
+
+@available(macOS 12, iOS 15, tvOS 15, watchOS 8, *)
```

Comment:
Is there anything stopping us from making it available in older systems? (same question applies to the `AsyncSequence` extension below)
`@available(macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0, *)`

Reply:
I'll check a bit later, thanks !

Reply:
Should be good now, wasn't entirely focused this noon :) Thanks for the great review!